### PR TITLE
Cigarette packs will brand their cigarettes.

### PR DESCRIPTION
**Pull request by xxalpha**

### CODE REVIEW

File: code/game/objects/items/weapons/storage/fancy.dm [hunk 1]
```diff
@@ -96,11 +96,14 @@
...
 	icon_type = "cigarette"
 	spawn_type = /obj/item/clothing/mask/cigarette
+	var/brand = "Space Cigarettes"
 
 /obj/item/weapon/storage/fancy/cigarettes/New()
 	..()
 	flags |= NOREACT
 	create_reagents(15 * storage_slots)//so people can inject cigarettes without opening a packet, now with being able to inject the whole one
+	for(var/obj/item/clothing/mask/cigarette/cig in src)
+		cig.desc = "A [brand] brand cigarette."
```
Comment (KorPhaeron) [inline code/game/objects/items/weapons/storage/fancy.dm]: Won't this end up reading "A unknown brand" or "A Uplift Smooth"?

Comment (xxalpha) [inline code/game/objects/items/weapons/storage/fancy.dm]: Sounds perfectly correct to me.

Comment (Iamgoofball) [inline code/game/objects/items/weapons/storage/fancy.dm]: >A Uplift Smooth
>sounds perfectly correct to me

AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA

Comment (xxalpha) [inline code/game/objects/items/weapons/storage/fancy.dm]: Sounds perfectly correct to myself.

File: code/game/objects/items/weapons/storage/fancy.dm [hunk 2]
```diff
@@ -181,11 +190,13 @@
 	desc = "You can't understand the runes, but the packet smells funny."
 	icon_state = "midori"
 	spawn_type = /obj/item/clothing/mask/cigarette/rollie
+	brand = "A Midori Tabako"
 
 /obj/item/weapon/storage/fancy/cigarettes/cigpack_shadyjims
 	name ="\improper Shady Jim's Super Slims"
 	desc = "Is your weight slowing you down? Having trouble running away from gravitational singularities? Can't stop stuffing your mouth? Smoke Shady Jim's Super Slims and watch all that fat burn away. Guaranteed results!"
```
Comment (TrustyGun) [inline code/game/objects/items/weapons/storage/fancy.dm]: Won't the cig.desc override this one?

Comment (xxalpha) [inline code/game/objects/items/weapons/storage/fancy.dm]: No.

**Comment by bgobandit** [comment]
how does this work wtih syndie cigarettes

File: code/game/objects/items/weapons/storage/fancy.dm (hunk 2 above)
Comment (TrustyGun) [inline code/game/objects/items/weapons/storage/fancy.dm]: oh wait i'm stupid nvm

**Comment by Jordie0608** [comment]
This could totally be done without a new var by manipulating the name into cig.desc

**Comment by xxalpha** [comment]
It can't.

**Comment by Jordie0608** [comment]
name = "DromedaryCo"
cig.desc = "\An [name]"
name = "\improper [name] packet"

**Comment by xxalpha** [comment]
Well that way it can.

**Comment by Jordie0608** [comment]
I should've posted that at first sorry; a single var doesn't matter too much however so I guess only if you can be bothered

**Comment by xxalpha** [comment]
The syndie cigs will show as "A cigarette packet brand cigarette" though.

**Comment by xxalpha** [comment]
What's the \An supposed to do?

**Comment by Jordie0608** [comment]
I think it inserts either 'A' or 'An' where appropriate, but I'm not sure if it actually does

**Comment by xxalpha** [comment]
Yeah it doesn't work.

**Pull request merged**